macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -441,6 +440,7 @@
 					"$(inherited)",
 					"@executable_path/../Frameworks",
 				);
+				PRODUCT_BUNDLE_IDENTIFIER = com.leastauthority.destiny;
```

Comment:
pubspec.yaml contains: identity_name: com.leastauthority.Destiny, should we fix that to have low case?
Also in code we have: com.leastauthority.dartWormholeGui and com.leastauthority.dart_wormhole_gui (for Android, which we complicated to change, probably) :)

Reply:
Renaming for the other parts has been done for consistency in #155